src/owlet.js

```diff
@@ -188,10 +188,13 @@ export class OwletEditor {
     }
 
     async setState(state) {
+        // Turn invisible characters into equivalent visible ones.
         // eslint-disable-next-line no-control-regex
-        const basic = state.program.replace(/[\x00-\x09\x0b-\x1f\x7f-\u009f]/g, function (c) {
+        let basic = state.program.replace(/[\x00-\x09\x0b-\x1f\x7f-\u009f]/g, function (c) {
             return String.fromCharCode(c.charCodeAt(0) | 0x100);
         });
+        // BBCMicrobot normalises smartquotes like this.
+        basic = basic.replace(/[\u201c\u201d]/g, '"');
```

Comment:
```suggestion
        const basic = state.program.replace(/[\x00-\x09\x0b-\x1f\x7f-\u009f]/g, function (c) {
            return String.fromCharCode(c.charCodeAt(0) | 0x100);
        }).replace(/[\u201c\u201d]/g, '"'); // BBCMicrobot normalises smartquotes like this.
```

Reply:
(might be a bit much but it avoids mutable variables when we don't really need them)

Reply:
Works for me (I even wondered about that approach originally).